.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Thank you for adding this and `.github/workflows/dependency-review.yml` there is an on-going PR where this files will be added alongside with SCA GHA.